plugins/noauth/fps_noauth/models.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+
+
+class BaseUser(BaseModel):
+    username: str = ""
```

Comment:
Why is this needed?

Reply:
We use it here https://github.com/jupyter-server/jupyverse/blob/20ee03b6bc3bb8f9b46bc09a47a44fd772796b46/plugins/noauth/fps_noauth/backends.py#L36

Reply:
But should we? Maybe `noauth` should not depend on a user:
```py
async def update_user(user):
```

Reply:
Probably not. But jupyverse still expects a user.
https://github.com/jupyter-server/jupyverse/blob/20ee03b6bc3bb8f9b46bc09a47a44fd772796b46/plugins/noauth/pyproject.toml#L34

Reply:
This can probably be an empty class:
```
class User:
    pass
```